server-phoenix/lib/helios_web/router.ex

```diff
@@ -22,7 +22,7 @@ defmodule HeliosWeb.Router do
 
   scope "/api" do
     pipe_through(:api)
-
+    forward "/graphql", Absinthe.Plug, schema: HeliosWeb.Schema
```

Comment:
I'd split this out of ` feat: add cors and graphql api endpoint `. `and` in generally smell that two things are happening in a commit. This is separate from cors work.